Firefox_Private_Network_ToS/en-US.md

```diff
@@ -0,0 +1,82 @@
+# Firefox Private Network Browser Extension (Beta) Terms of Service
+
+Version 3.0, effective July 7, 2020
+
+Firefox Private Network (“Service”) is an experimental offering. Please read these Terms of Service carefully because they explain important information about your use of the Service.
+
+If you activate this Service, it keeps your Firefox web traffic private from your internet service provider, and most websites you visit by encrypting and routing it through our partner’s networks instead. It will also replace your actual IP address to keep your personal IP address private from others, such as the websites you visit and your internet service provider.
+
+## You Must Be Eligible to Use the Service
+
+You must be in the United States to download and use the Service.
+
+A Firefox Account is required to use the Service. To create a Firefox Account, you will also need to agree to the [Terms of Service](https://www.mozilla.org/about/legal/terms/services/) and [Privacy Notice](https://www.mozilla.org/privacy/firefox/) for your Firefox Account.
+
+## Your Privacy
+
+__Your Privacy.__ The [Firefox Private Network Privacy Notice](https://www.mozilla.org/privacy/firefox-private-network/) explains what information is sent when you use the Service and how we handle and share that information.
+
+Mozilla is a global organization, and our computers and service providers’ computers may be in various countries around the world, including the United States. This means that your information might be processed on servers located outside of the country where you live, and that country may have a different level of data protection regulation than yours. By giving us information, you consent to this kind of transfer of your information.
+
+## Permissions Necessary to Provide the Service
+
+Firefox Private Network is in partnership with Cloudflare. To learn more about how the Service works, you can see Mozilla’s [source code](https://github.com/mozilla/secure-proxy) for Firefox Private Network.
+
+__Your Use of the Service.__ You give Mozilla and Cloudflare all rights to operate the Service, including processing the data you upload to or access through the Service. You can find more information about the data Mozilla and Cloudflare process in the Firefox Private Network [Privacy Notice](https://www.mozilla.org/privacy/firefox-private-network/).
```

Comment:
Include the full FPN PN in the link: `[Firefox Private Network Privacy Notice](...), like in line 17.

Reply:
Updated